math/min/min_test.go

```diff
@@ -1,81 +1,62 @@
-package min_test
+package min
 
 import (
 	"testing"
-
-	"github.com/TheAlgorithms/Go/math/min"
 )
 
-func TestMin(t *testing.T) {
-	testCases := []struct {
-		name  string
-		left  int
-		right int
-		min   int
+func getTestCases() []struct {
+	name    string
+	base    int
+	numbers []int
+	min     int
+} {
+	var tests = []struct {
+		name    string
+		base    int
+		numbers []int
+		min     int
 	}{
-		{
-			name:  "left is min",
-			left:  1,
-			right: 10,
-			min:   1,
-		},
-		{
-			name:  "right is min",
-			left:  10,
-			right: 9,
-			min:   9,
-		},
+		{"BASE: 8bit, numbers: [128, 127], min = 117", 8, []int{128, 127}, 127},
```

Comment:
The names are only relevant to bitwise function and not other ones. I think you can just change these to 
```suggestion
		{"minimum of [128, 127]", 8, []int{128, 127}, 127},
```